docs/BUILD.md

```diff
@@ -106,6 +106,20 @@ The entire list of s2n-tls CMake options can be viewed with the following comman
 cmake . -LH
 ```
 
+### RAND engine override
+
```

Comment:
We expect this to be a niche use case. Let's not put it higher up than building with specific libcryptos.

It might need to go lower than other topics as well.